prebuild_binaries/build_scripts/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+arch_list=( x64 )
+node_versions=( 6.0.0 8.0.0 10.0.0 11.0.0 )
+
+cd $(dirname $0)
+cd $(dirname $0)/../..
```

Comment:
Having two "cd" commands seems weird? Remove the first one?

Reply:
Done.